### DISCUSSION TITLE
PortalSpawn

### DISCUSSION THREAD

**Discussion by Waaayne** [vrcx-team/VRCX#951] [Q&A]
Hey,
so this used to work, but now, whether it's a friend or not, if someone drops a portal, it doesn't show up in the Game Log anymore.
Is this a known bug or did something get messed up after the  2024.10.11 update?
Does anyone else have this problem?
Would appreciate the help :)

**Comment by Natsumi-sama** [comment]
VRChat removed logging for portals, filter only exists for old entries.
